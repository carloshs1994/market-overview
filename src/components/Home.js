import React from 'react';

const Home = () => (
  <main>
    <h1>Home</h1>
  </main>
);

export default Home;
