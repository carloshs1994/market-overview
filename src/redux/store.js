import { createStore, combineReducers, applyMiddleware } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';
import homeReducer from './home/home';
import detailsReducer from './details/details';

const reducers = combineReducers({
  homeReducer,
  detailsReducer,
});

const store = createStore(
  reducers,
  applyMiddleware(logger, thunk),
);

export default store;
