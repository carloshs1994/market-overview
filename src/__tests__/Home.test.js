import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Home from '../components/Home';
import store from '../redux/store';

test('matches voice search component snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <Home />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
