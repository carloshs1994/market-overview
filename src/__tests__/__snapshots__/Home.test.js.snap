// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches voice search component snapshot 1`] = `
<main>
  <section
    className="search"
  >
    <form
      onSubmit={[Function]}
    >
      <label
        htmlFor="stock-name"
      >
        <input
          id="stock-name"
          name="stockName"
          onChange={[Function]}
          placeholder="Search name or symbol"
          required={true}
          type="text"
          value=""
        />
      </label>
      <button
        aria-label="search"
        type="submit"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            Object {
              "color": "white",
              "fontSize": "1.2rem",
            }
          }
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"
          />
        </svg>
      </button>
    </form>
  </section>
  <section
    className="description"
  >
    <p>
      Showing the 0 Most Active Stocks
    </p>
  </section>
  <ul>
    <li
      style={
        Object {
          "width": "100%",
        }
      }
    >
      <p
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        No Match :(
      </p>
    </li>
  </ul>
</main>
`;
